documentatie/gebruikersdocumentatie/verkiezingsproces-en-abacus.md

```diff
@@ -110,20 +108,21 @@ In de afbeelding hieronder zie je hoe Abacus werkt op het gemeentelijk stemburea
 
 ![Uitslagvaststelling](/documentatie/gebruikersdocumentatie/img/proces%20uitslagvaststelling.png)
 
-- IMPORT: gebruikers kunnen kandidaten en verkiezingsbestanden in EML-formaat importeren in Abacus.
-- INVOER: gebruikers kunnen verkiezingsresultaten invoeren, inclusief verklaringen over ontbrekende stembiljetten/stempassen en klachten van kiezers.
+- Bovenaan staat IMPORT: gebruikers kunnen kandidaten en verkiezingsdefinities in EML-formaat importeren in Abacus.
+- Links staat INVOER: gebruikers kunnen verkiezingsresultaten invoeren, inclusief verklaringen over ontbrekende stembiljetten/stempassen en klachten van kiezers.
 Abacus telt de resultaten op en controleert ze.
-De data-analyse leidt tot waarschuwingen voor kleine afwijkingen en foutmeldingen voor grote afwijkingen en inconsistente resultaten.
-- ONDERTEKENING: de data wordt ondertekend met een handtekening van de voorzitter van het gemeentelijk stembureau.
-- EXPORT: de verkiezingsresultaten worden geëxporteerd als EML_NL-bestand en als PDF-bestand.
+- Rechts staat ONDERTEKENING: de data wordt ondertekend met een handtekening van de voorzitter van het gemeentelijk stembureau.
+- Onderaan staat EXPORT: de verkiezingsresultaten worden geëxporteerd als EML_NL-bestand en als PDF-bestand.
 
 ### Technische keuzes en architectuur
 
-De software wordt gebruikt via de webbrowser, en dit doen we met HTML, CSS en JavaScript. Hiervoor gebruiken we TypeScript, een meer uitgebreide versie van JavaScript met typing. Voor de gebruikersinterface gebruiken we de populaire library React.
+Abacus zal worden gedraaid op een gesloten netwerk zonder internettoegang. De backend draait op één machine en kan via de webbrowser (frontend) worden benaderd vanaf andere computers op het netwerk.
 
-Voor de backend gebruiken we Rust, een high-level programmeertaal waarmee je efficiënt kunt programmeren. Ook heeft deze taal een 'strict type system' waardoor minder bugs ontstaan. Ook is dit een populaire, open-source programmeertaal met een grote community.
+Voor de frontend wordt TypeScript gebruikt, een meer uitgebreide versie van JavaScript met typing. Voor de gebruikersinterface wordt de populaire library React gebruikt.
 
-Voor de database gebruiken we SQLite. Deze library is populair, lichtgewicht en gebruiksvriendelijk, en bovendien is installatie van SQLite niet vereist.
+De backend wordt gemaakt met Rust, een high-level programmeertaal waarmee je efficiënt kunt programmeren. Het unieke eigendomssysteem van Rust voorkomt geheugenfouten, en de taal heeft een 'strict type system' waardoor minder bugs ontstaan. Ook is dit een populaire, open-source programmeertaal met een grote community.
```

Comment:
"unieke eigendomssysteem". Is dit een vertaling van een engels begrip? Ik ben redelijk ingevoerd maar snap niet wat hier staat. Kan me voorstellen dat het voor leken nog meer abacadabra is.

Reply:
Dat klopt, het is een vrij letterlijke vertaling van het concept 'ownership'. Ik heb het teruggezet naar het Engels en een linkje toegevoegd naar de documentatie van Rust voor de mensen die willen weten hoe het werkt. Dit heb ik ook gedaan voor 'strict type system'.

```suggestion
De backend wordt gemaakt met Rust, een high-level programmeertaal waarmee je efficiënt kunt programmeren. Het unieke systeem van '[ownership](https://doc.rust-lang.org/book/ch04-01-what-is-ownership.html)' in Rust voorkomt geheugenfouten, en de taal heeft een '[strict type system](https://doc.rust-lang.org/book/ch03-02-data-types.html)' waardoor minder bugs ontstaan. Ook is dit een populaire, open-source programmeertaal met een grote community.
```

Reply:
Top, prima suggestie. Commit jij 'm?